docker-entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
I would recommend making a separate folder for scripts that are only used within the  built image.